filer/fields/folder.py

```diff
@@ -100,6 +97,7 @@ def __init__(self, rel, queryset, to_field_name, *args, **kwargs):
         self.max_value = None
         self.min_value = None
         kwargs.pop('widget', None)
+        kwargs.pop('blank', None)
```

Comment:
@yakky I'm not sure if this is the right approach, in 3.1 we get the following error otherwise while using djangocms-file:
  File "/Users/finalangel/Sites/_testing/djangocms380/mysite/packages/djangocms-file/djangocms_file/cms_plugins.py", line 36, in <module>
    class FolderPlugin(CMSPluginBase):
  File "/Users/finalangel/Sites/_testing/djangocms380/mysite/packages/django-cms/cms/plugin_base.py", line 61, in __new__
    new_plugin.form = type('%sForm' % name, (forms.ModelForm,), form_attrs)
  File "/Users/finalangel/.pyenv/versions/django-cms-3.8.0/lib/python3.6/site-packages/django/forms/models.py", line 258, in __new__
    apply_limit_choices_to=False,
  File "/Users/finalangel/.pyenv/versions/django-cms-3.8.0/lib/python3.6/site-packages/django/forms/models.py", line 175, in fields_for_model
    formfield = f.formfield(**kwargs)
  File "/Users/finalangel/Sites/_testing/djangocms380/mysite/packages/django-filer/filer/fields/folder.py", line 135, in formfield
    return super().formfield(**defaults)
  File "/Users/finalangel/.pyenv/versions/django-cms-3.8.0/lib/python3.6/site-packages/django/db/models/fields/related.py", line 994, in formfield
    'blank': self.blank,
  File "/Users/finalangel/.pyenv/versions/django-cms-3.8.0/lib/python3.6/site-packages/django/db/models/fields/related.py", line 419, in formfield
    return super().formfield(**defaults)
  File "/Users/finalangel/.pyenv/versions/django-cms-3.8.0/lib/python3.6/site-packages/django/db/models/fields/__init__.py", line 929, in formfield
    return form_class(**defaults)
  File "/Users/finalangel/Sites/_testing/djangocms380/mysite/packages/django-filer/filer/fields/folder.py", line 100, in __init__
    forms.Field.__init__(self, widget=self.widget(rel, site), *args, **kwargs)
TypeError: __init__() got an unexpected keyword argument 'blank'
```

```

Reply:
*forms* `Field` does not have the `blank` attribute, I don't get why it's passed to `AdminFolderFormField`

Reply:
@yakky I'm also getting that issue on other addons. It seems like something changed from Django<3.1 to 3.1. I need to add that also on the CMS. Let me know if you figure out more eventually.

Reply:
I hope to be able to check this in a couple of days

as a stop gap solution, we can keep the current implementation, but I agree that it might there be more behind it